.github/workflows/e2e-test.yml

```diff
@@ -0,0 +1,46 @@
+name: Local Env CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  local-env:
+    name: Setup Local Environment
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout Local Env repository
+      uses: actions/checkout@v4
+
+    - name: Setup and Start Local Env Dev Container
+      uses: devcontainers/ci@v0.3
+      with:
+        runCmd: |
+          kind create cluster --name to-dos --config kind-local-config.yaml --kubeconfig ./.to-dos-cluster-kubeconfig
+          helmfile cache cleanup && helmfile --environment local --namespace local -f deploy/helmfile.yaml apply
+
+    - name: Checkout UI repository
+      uses: actions/checkout@v4
+      with:
+        repository: FaceMan01/to-dos-ui
+        path: to-dos-ui
+
+    - name: Setup and Start UI Dev Container
+      uses: devcontainers/ci@v0.3
+      with:
+        configFile: ./to-dos-ui/.devcontainer/devcontainer.json
+        runCmd: |
+          echo "127.0.0.1       to-dos.local.tourmalinecore.internal" >> /etc/hosts
+          curl http://to-dos.local.tourmalinecore.internal:40080/api/to-dos-api/to-dos/
+          cd to-dos-ui
+          npm ci
+          npm audit fix
+          npm run start-local-env &
```

Comment:
Is it actually needed to run UI here? Shouldn't we expect that it is already running in the local-env?

Reply:
In this situation, if you do not start the UI server, there will be an error:
Cypress could not verify that this server is running:
> http://localhost:5173

Reply:
Can you please send the error log?
BTW why do you make npm audit fix? Doing that you are changing the deps which might lead to a different behavior comparing to just npm ci.

Reply:
I wrote npm audit fix because I did it when I was running it locally and it worked correctly. If you don't start the server, there will be an error anyway. It does't depends on npm audit fix, I've tested. The log is presented below. You can also view it in [Actions](https://github.com/FaceMan01/to-dos-local-env/actions/runs/12212665719/job/34071378839).
```
Cypress could not verify that this server is running:
    > http://localhost:5173
We are verifying this server because it has been configured as your baseUrl.
Cypress automatically waits until your server is accessible before running tests.
    
Cypress failed to verify that your server is running. 
Please start this server and then run Cypress again.
```

Reply:
I have fixed it